exporter/elasticsearchexporter/config.go

```diff
@@ -224,6 +225,8 @@ func (m MappingMode) String() string {
 		return "otel"
 	case MappingRaw:
 		return "raw"
+	case MappingBodyMap:
+		return "bodymap"
```

Comment:
Not a blocker for this PR but I think what we're missing is an `auto` mode that chooses the right mapping mode based on metadata for a batch. In hosted environments, there's just one OTLP receiver but the data may be mapped either in `otel` or `bodymap` mode.